src/styles/_catalog.scss

```diff
@@ -14,6 +13,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    &:hover {
+      cursor: pointer;
+      color: #34568b;
+      transition: transform 300ms;
+      transform: scale(1.2);
```

Comment:
transition should be added to default state, not hovered, and color change isn't animated now